package.json

```diff
@@ -11,9 +11,11 @@
     "e2e": "node test/e2e/runner.js",
     "test": "npm run unit && npm run e2e",
     "lint": "eslint --ext .js,.vue src test/unit/specs test/e2e/specs",
-    "build": "node build/build.js"
+    "build": "node build/build.js",
+    "vtest": "backstop reference && backstop test || true"
```

Comment:
Looking into the backstop code, it looks like maybe we only run the `backstop reference` on the first build, and on subsquent builds we should be running `backstop test` only to compare against those references? And then once we're set, we run `backstop approve`? So maybe we should add three methods here:

`vtest-init`: Run the above code and build our references
`vtest`: Just runs the `backstop test` code
`vtest-approve`: Runs `backstop approve`

Or we scan just change this to `vtest-init` and change the readme to run `vtest-init` followed by `backstop test` and `backstop approve`?